tests/mock_tests/aclorch_ut.cpp

```diff
@@ -628,16 +628,74 @@ namespace aclorch_test
         // consistency validation with CRM
         bool validateResourceCountWithCrm(const AclOrch *aclOrch, CrmOrch *crmOrch)
         {
-            auto resourceMap = Portal::CrmOrchInternal::getResourceMap(crmOrch);
-            auto ifpPortKey = Portal::CrmOrchInternal::getCrmAclKey(crmOrch, SAI_ACL_STAGE_INGRESS, SAI_ACL_BIND_POINT_TYPE_PORT);
-            auto efpPortKey = Portal::CrmOrchInternal::getCrmAclKey(crmOrch, SAI_ACL_STAGE_EGRESS, SAI_ACL_BIND_POINT_TYPE_PORT);
+             // ACL counter
+            {
+                auto const &resourceMap = Portal::CrmOrchInternal::getResourceMap(crmOrch);
+                uint32_t crm_acl_cnt = 0;
+                for (auto const &kv : resourceMap.at(CrmResourceType::CRM_ACL_TABLE).countersMap)
+                {
+                    crm_acl_cnt += kv.second.usedCounter;
+                }
 
-            auto ifpPortAclTableCount = resourceMap.at(CrmResourceType::CRM_ACL_TABLE).countersMap[ifpPortKey].usedCounter;
-            auto efpPortAclTableCount = resourceMap.at(CrmResourceType::CRM_ACL_TABLE).countersMap[efpPortKey].usedCounter;
+                if (crm_acl_cnt != Portal::AclOrchInternal::getAclTables(aclOrch).size())
```

Comment:
crm_acl_cnt isn't very clear. Could you rename this to something more similar to the old tests so that it's clear you're checking the number of ACL tables?

Reply:
change to crm_acl_table_cnt